### PR TITLE
Added TSG for security vulnerability CVE 2024-49060

### PR DESCRIPTION
**Pull request by alkohli**

### CODE REVIEW

File: TSG/ArcVMs/security-vulnerability-cve-2024-49060.md [hunk 1]
```diff
@@ -0,0 +1,147 @@
+# ISO vulnerability for Arc VMs running on Azure Stack HCI, version 23H2
+
+## Description
+
+Microsoft has identified a security vulnerability that could expose the local admin credentials used during the creation of Arc VMs on Azure Stack HCI to non-admin users on the VM and on the hosts.  
+
+## Affected releases
+
+Arc VMs running releases prior to Azure Stack HCI, 2411 release are vulnerable.
```
Comment (kerimhanif) [inline TSG/ArcVMs/security-vulnerability-cve-2024-49060.md:9]: Do we have the comma in the right place here?

File: TSG/ArcVMs/security-vulnerability-cve-2024-49060.md [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+
+## Mitigation
+
+To mitigate this risk, we recommend that you rotate the passwords on the accounts used when creating Arc VMs on Azure Stack HCI. Completing this action will immediately protect your environment.  
+
+## Ongoing protection
+
+To ensure ongoing protection for newly provisioned VMs, update to the latest 2411 release. This release includes a fix that prevents this vulnerability for newly provisioned VMs, eliminating the need for additional password rotation.
```
Comment (kerimhanif) [inline TSG/ArcVMs/security-vulnerability-cve-2024-49060.md:17]: Let's change "This release includes a fix" to "This release includes fixes"

File: TSG/ArcVMs/security-vulnerability-cve-2024-49060.md [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+
+## Mitigation
+
+To mitigate this risk, we recommend that you rotate the passwords on the accounts used when creating Arc VMs on Azure Stack HCI. Completing this action will immediately protect your environment.  
+
+## Ongoing protection
+
+To ensure ongoing protection for newly provisioned VMs, update to the latest 2411 release. This release includes a fix that prevents this vulnerability for newly provisioned VMs, eliminating the need for additional password rotation.  
+
+To mitigate this vulnerability, first identify all the Arc VMs created before the Azure Stack HCI, 2411 release. Then, change the account password on each VM.
```
Comment (kerimhanif) [inline TSG/ArcVMs/security-vulnerability-cve-2024-49060.md:19]: Again the comma sounds better here but still is it correctly placed?

File: TSG/ArcVMs/security-vulnerability-cve-2024-49060.md [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+
+### Identify Arc VMs
+
+1. Identify all the Arc VMs created before the Azure Stack HCI, 2411 release, run the following script. Make sure to change the variables in the beginning with the appropriate values.
```
Comment (kerimhanif) [inline TSG/ArcVMs/security-vulnerability-cve-2024-49060.md:23]: "the variables in the beginning" to "the variables in the beginning of the script"

File: TSG/ArcVMs/security-vulnerability-cve-2024-49060.md [hunk 5]
```diff
@@ -0,0 +1,147 @@
...
+    #Login to the Azure Tenant 
+    az login --tenant $tenantID 
+
+    # Get the update date of the cluster  
+    $install_date=Get-AzResource -ResourceId "/subscriptions/$subscriptionID/resourceGroups/$resourceGroup/providers/microsoft.azurestackhci/clusters/$clusterName/updates/Solution10.2408.2.7" -ExpandProperties 
+
+    # Get the list of VMs and their creation dates 
+    $vms=az stack-hci-vm list --resource-group $resourceGroup | ConvertFrom-json 
+
+    # Format the table and flag VMs that are updated before the update date
```
Comment (kerimhanif) [inline TSG/ArcVMs/security-vulnerability-cve-2024-49060.md:41]: "flag VMs that are" to "flag VMs that were"

**Review by kerimhanif — CHANGES_REQUESTED** [review]
Added some comments and change requests

**Review by charlesjoy — APPROVED** [review]

**Pull request merged**